*** runConfigure.original	Wed Sep  7 08:55:53 2005
--- runConfigure	Thu Feb 24 17:49:34 2011
***************
*** 431,438 ****
      if test $platform; then
          case $platform in
             solaris)
!               bitstobuildDefines=" $bitstobuildDefines -xarch=v9 "
!               bitstobuildLink=" -xarch=v9 " ;;
             aix)
                bitstobuildDefines=" $bitstobuildDefines -q64 -qwarn64 " ;;
             irix)
--- 431,438 ----
      if test $platform; then
          case $platform in
             solaris)
!               bitstobuildDefines=" $bitstobuildDefines -m64 "
!               bitstobuildLink=" -m64 " ;;
             aix)
                bitstobuildDefines=" $bitstobuildDefines -q64 -qwarn64 " ;;
             irix)
